packages/wonder-blocks-modal/src/components/modal-dialog.tsx

```diff
@@ -44,9 +44,10 @@ type Props = {
      */
     testId?: string;
     /**
-     * The ID of the title labelling this dialog, if applicable.
+     * The ID of the title labelling this dialog. Required.
+     * See WCAG 2.1: 4.1.2 Name, Role, Value
      */
```

Comment:
**suggestion:** Could you also please add this comment to the argtypes file? tbh it's a bit annoying having to repeat this, but Storybook Docs has some limitations around that.

https://github.com/Khan/wonder-blocks/blob/main/__docs__/wonder-blocks-modal/modal-dialog.argtypes.tsx#L55-L62